CHANGELOG.md

```diff
@@ -49,39 +55,30 @@ This release is intended for the PureScript 0.9.1 compiler and newer.
 
 **Note**: The v1.0.0 tag is not meant to indicate the library is “finished”, the core libraries are all being bumped to this for the 0.9 compiler release so as to use semver more correctly.
 
-## [v1.0.0-rc.1](https://github.com/purescript/purescript-validation/releases/tag/v1.0.0-rc.1) - 2016-03-24
```

Comment:
I've removed the release candidates as we don't usually include these in changelogs.